service/src/main/java/com/codeforcommunity/processor/ProtectedSiteProcessorImpl.java

```diff
@@ -915,4 +922,24 @@ public void approveSiteImage(JWTData userData, int imageID) {
     imageRecord.setApprovalStatus(ImageApprovalStatus.APPROVED.getApprovalStatus());
     imageRecord.store();
   }
+
+  @Override
+  public void rejectSiteImage(JWTData userData, int imageId, RejectImageRequest rejectImageRequest) {
+    checkImageExists(imageId);
+    assertAdminOrSuperAdmin(userData.getPrivilegeLevel());
+
+    if (rejectImageRequest.getRejectionReason() != null) {
+      int uploaderId  = db.select(SITE_IMAGES.UPLOADER_ID)
+                      .from(SITE_IMAGES)
+                      .where(SITE_IMAGES.ID.eq(imageId))
+                      .fetchOne(0, int.class);
+
+      UsersRecord user = db.selectFrom(USERS).where(USERS.ID.eq(uploaderId)).fetchOne();
+      String userEmail = user.getEmail();
+      String userFullName =
+              AuthDatabaseOperations.getFullName(user.into(Users.class));
+      emailer.sendRejectImageEmail(userEmail, userFullName, rejectImageRequest.getRejectionReason());
+    }
```

Comment:
Do we want to notify people that their image was rejected/deleted even if a reason isn't provided?

Reply:
I guess yes (https://github.com/Code-4-Community/speak-for-the-trees-backend-v2/pull/222#discussion_r1485656681)